### PR TITLE
Walking Skeleton

### PR DESCRIPTION
**Pull request by kelseyroy**
[Walking Skeleton](https://github.com/8thlight/apprenticeship_syllabus/blob/master/shared_resources/projects/tic_tac_toe/01_beginner/walking_skeleton.md)

**What's changed:**
- Generated Gemfile using `bundle init`
- Added RSpec gem to Gemfile and ran bundle to install RSpec gem
- Initialized RSpec within the project to create `.rspec` and `spec/spec_helper.rb`
- Created basic RSpec test that affirms true is true.
- Defined local Ruby version with [asdf](https://asdf-vm.com/) by executing `asdf local ruby latest`
- Created bin folder to house executable script from command line. `play.rb ` instantiates a new game and posts welcome message.
- Set up YAML file

**MISC**
- Created `lib` folder to house future code
- Created `.gitignore` file and populated using [GitHub's Ruby.gitignore template](https://github.com/github/gitignore/blob/main/Ruby.gitignore)

**Still To Do:**
- [x] Write one passing test
- [x] Write README
- [x] Write Welcome Message
- [x] Set up CI

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+
+gem 'rspec'
+# gem 'rubocop', '~> 1.48', '>= 1.48.1'
+# bundle with rspec on YAML file
+# gem "rails"
```
Comment (Aoxian) [inline Gemfile]: If you are not going to use it, just get rid of it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+## How to play
+
+Tic Tac Toe is a two player game played on a 3 x 3 grid. One player will mark the grid with 'X' and the other player will mark the grid with 'O'. Each player takes turns placing their mark in the squares on the grid, with the goal of getting 3 marks in a row.
+
+## Setup
+1. [Clone the repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) to your local computer.
+2. Use your terminal to navigate into your new tic-tac-toe folder and install project dependencies by running <code>bundle install</code>.
+3. To start the game, try running <code>ruby bin/play.rb</code> from within the tic-tac-toe folder. 
+4. To run the tests, try running  <code>bundle exec rspec</code> from within the tic-tac-toe folder.
+
```
Comment (Aoxian) [inline README.md]: "Do or do not, there is no 'try'" - Yoda.

**Review by Aoxian — APPROVED** [review]
Misc. feedback, nothing blocking.

**Review by ghost — APPROVED** [review]
Looks Good!

**Pull request merged**